OpenSim/Tools/InverseDynamicsTool.cpp

```diff
@@ -367,13 +367,18 @@ bool InverseDynamicsTool::run()
         SpatialVec equivalentBodyForceAtJoint;
 
         for(int i=0; i<nt; i++){
-            StateVector genForceVec(times[i], nq, &((genForceTraj[i])[0]));
+            StateVector
+                genForceVec(times[i],
+                            SimTK::Vector_<double>(nq,
+                                                   &((genForceTraj[i])[0])));
```

Comment:
`genForceTraj` is a `SimTK::Array_<Vector>`. It is defined [here](https://github.com/opensim-org/opensim-core/blob/a245657272f83a4976cab9662e1316fb91bed282/OpenSim/Tools/InverseDynamicsTool.cpp#L330).